App/Talent.App.WebApp/wwwroot/js/react/ReactScripts/Employer/ManageJob/JobSummaryCard.jsx

```diff
@@ -1,19 +1,126 @@
 ﻿import React from 'react';
 import Cookies from 'js-cookie';
-import { Popup } from 'semantic-ui-react';
+import { CardMeta,
+    CardHeader,
+    CardGroup,
+    CardDescription,
+    CardContent,
+    Button,
+    Card,
+    Icon,
+    Popup } from 'semantic-ui-react';
 import moment from 'moment';
+import { Link } from 'react-router-dom';
 
 export class JobSummaryCard extends React.Component {
     constructor(props) {
         super(props);
-        this.selectJob = this.selectJob.bind(this)
+        this.selectJob = this.selectJob.bind(this);
+
+        this.state = {
+            jobs: this.props.jobData,
+            cards: []
+        };
+    }
+
+    convertLocation(location) {
+        let locationMeta = '';
+        if (location) {
+            let { city, country } = location;
+            if (city && country) {
+                locationMeta = `${city}, ${country}`;
+            } else if (city) {
+                locationMeta = city;
+            } else if (country) {
+                locationMeta = country;
+            }
+        }
+
+        return locationMeta;
+    }
+
+    init() {
+        this.props.jobData.forEach(job => {
+            let card = (
+                <Card key={job.id} fluid >
+                    <CardContent>
+                        <CardHeader>{job.title}</CardHeader>
+                        <CardMeta>{this.convertLocation(job.location)}</CardMeta>
+                        <CardDescription>{job.summary}</CardDescription>
+                    </CardContent>
+                    <CardContent extra>
+                        <div className='ui left floated'>
+                            {                                
+                                moment(job.expiryDate).isAfter(moment()) ? 
+                                    <Button content='Active' color='green' /> : 
+                                    <Button content='Expired' color='red' />
+                            }     
+                        </div>
+                        <div className='ui right floated'>
+                            <Button basic color='blue' onClick={() => this.closeJob(job.id)}>
+                                <Icon name='ban' />
+                                Close
+                            </Button>
+                            <Button basic color='blue' as={Link} to={`/EditJob/${job.id}`}>
+                                <Icon name='edit' />
+                                Edit
+                            </Button>
+                            <Button basic color='blue' as={Link} to={`/PostJob/${job.id}`}>
+                                <Icon name='copy' />
+                                Copy
+                            </Button>
+                        </div>
+                    </CardContent>
+                </Card>
+            );
+
+            this.state.cards.push(card);
+        });
```

Comment:
Before directly displaying this, can you first verify if this.props.jobData has items or not? There's nothing wrong with your code, but you need to check them first because you might encounter a scenario wherein the API changes and instead of returning array the data returns an object. If that happens, your system will display an error so it is better to check and verify first.

Another thing, can you explain why are you storing the elements in a state instead of rendering them directly? Let's say you have 1000 items to handle. It wouldn;t be good if you will store all of them to a state.